package.json

```diff
@@ -28,6 +28,13 @@
     "url": "https://github.com/windbreaker-io/windbreaker-service-util/issues"
   },
   "homepage": "https://github.com/windbreaker-io/windbreaker-service-util#readme",
+  "config": {
+    "branchNameCommitModifier": {
+      "preChar": "[",
```

Comment:
We don't actually need `preChar` and `postChar`. Those are the defaults in `branch-name-commit-modifier`